Dockerfile

```diff
@@ -120,6 +123,18 @@ FROM base as dev
 # Swap user, so the following tasks can be run as root
 USER root
 
+# Install node (Keep the version in sync with the node container above)
+# Download and import the Nodesource GPG key
+# This step is necessary as the previous method of using setup_18.x is now deprecated.
```

Comment:
suggestion: Remove this last comment line as it requires knowledge of previous configurations to make sense.

Reply:
Done